man-roxygen/occtemp.r

```diff
@@ -2,7 +2,8 @@
 #' in each data source we query. Note: ebird now expects species codes instead of 
 #' scientific names - we pass you name through [rebird::species_code()] internally
 #' @param from (character) Data source to get data from, any combination of gbif, bison,
-#' inat, ebird, and/or vertnet
+#' inat, ebird, vertnet, idigbio, obis, or ala. See `vignette(topic = 'spocc introduction') 
```

Comment:
```suggestion
#' inat, ebird, vertnet, idigbio, obis, or ala. See the [spocc introduction](https://docs.ropensci.org/spocc/articles/spocc.html).
```

`vignette(topic = 'spocc introduction')` doesn't actually work for me, but it should, shouldn't it?